WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -149,15 +152,24 @@ protected void onCreate(Bundle savedInstanceState) {
                     }
                     break;
                 case SELFHOSTED_ONLY:
+                    mUnifiedLoginTracker.setSource(Source.SELF_HOSTED);
```

Comment:
I know we're reusing the names that are already being used by the application here, but I just wanted to point out that `SELF_HOSTED` is not an accurate name for this source, since it's possible to add non self-hosted sites from it as well. 

I've used "Add Site" before to describe this source, but I don't think it's a good name as well. Besides that, an argument could be made that changing this would cause even more confusion, so I'll leave it up to you to decide what's the better alternative here.